members/WIL000339.yaml

```diff
@@ -3,41 +3,43 @@ contact_form:
   method: post
   action: Contact.aspx
   steps:
-    - visit: "http://legis.wisconsin.gov/senate/miller/services/Pages/Email-Me.aspx"
+    - visit: "http://legis.wisconsin.gov/senate/16/miller/constituent-services/"
+    - find:
+        - selector: input[id='556ffd44-9d1a-472c-8bbb-375de761904a']
     - fill_in:
         - name: email
-          selector: "#legis-cf-email"
+          selector: input[id='556ffd44-9d1a-472c-8bbb-375de761904a']
           value: $EMAIL
           required: true
         - name: name
-          selector: "#legis-cf-name"
+          selector: input[id='d3c70554-f566-4fc3-8bd7-e750b91bd87d']
           value: $NAME_FULL
           required: true
         - name: address
-          selector: "#legis-cf-address"
+          selector: input[id='4843fa7c-8c6e-4622-bcdb-f035090ec7a1']
           value: $ADDRESS_STREET
           required: true
         - name: citystatezip
-          selector: "#legis-cf-city-state"
+          selector: input[id='4c66f7e4-536b-422d-a29c-90369ee6d6f8']
           value: $ADDRESS_CITY_STATE_ZIP
           required: true
-        - name: subject
-          selector: "#legis-cf-phone"
+        - name: Phone
+          selector: input[id='bdb4a01e-782e-44a6-fb4e-7d719b07e2dc']
           value: $PHONE
-          required: true
+          required: false
         - name: subject
-          selector: "#legis-cf-subject"
+          selector: input[id='27202a8c-46e6-452c-9bdc-3f1526130365']
           value: $SUBJECT
-          required: true
+          required: false
         - name: message
-          selector: "#legis-cf-message"
+          selector: textarea[id='40e41500-46c2-4209-af8c-2765c3fd427f']
           value: $MESSAGE
           required: true
     - click_on:
         - value: Submit
-          selector: ".legis-email-submit-btn"
+          selector: input[type='submit'][value='Submit']
   success:
     headers:
       status: 200
     body:
-      contains: Sign up for
\ No newline at end of file
+      contains: Thank you
```

Comment:
does `Thank you` appear anywhere on the form itself? If so there's a chance this causes false positives in terms of detecting successful submissions. Can this success message be made any more unique?